service/matching/handler/engine_integration_test.go

```diff
@@ -363,6 +364,7 @@ func (s *matchingEngineSuite) PollForTasksEmptyResultTest(callContext context.Co
 			Identity:   identity,
 		}
 		pollResp, err := pollTask(s.matchingEngine, s.handlerContext, pollReq)
+		pollResp.AutoConfigHint = nil // poller wait time is not a fixed value, exclude it from comparison
```

Comment:
How about adding additional assertion on NotNil on the autoconfighint?

Reply:
Added